nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'Subclasses must implement the correct_name method.'
+  end
```

Comment:
[OPTIONAL]
- [ ] It would be great to implement the error message dynamically as you may notice you hard coded it.

You may consider saying:
```rb
def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
```